package.json

```diff
@@ -23,7 +24,7 @@
     "webpack": "^1.13.1",
     "webpack-dev-server": "^1.14.1",
     "webpack-stream": "^3.2.0",
-    "webpack-svgstore-plugin": "^2.2.2",
+    "webpack-svgstore-plugin": "^3.0.0",
```

Comment:
I couldn't make it work with 3.0.0. I think you should revert this to 2.2.2 for now.